invenio_cli/helpers/commands.py

```diff
@@ -151,8 +151,25 @@ def _ensure_containers_running(self):
         docker_helper.start_containers()
         # TODO: Find faster way to procede when containers are ready
         time.sleep(30)  # Give time to the containers to start properly
+    
+    def destroy(self):
+        """Destroys env, containers."""
+        command = [
+            'pipenv', '--rm'
+            ]
+        subprocess.run(command, check=True)
+        click.secho('Destroyed Venv...', fg='green')
```

Comment:
Even though we use shortenings when talking among us, I guess for users is best to be explicit:
```suggestion
        click.secho('Virtual environment destroyed', fg='green')
```